package.json

```diff
@@ -13,5 +13,8 @@
     "eslint": "^7.21.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.22.1"
+  },
+  "dependencies": {
+    "luxon": "^1.26.0"
```

Comment:
No estáis usando este paquete para nada, ya lo estáis cargando con la etiqueta script.